### PR TITLE
Final review

### PR DESCRIPTION
**Pull request by Marcraphael12**
In this PR, we are submitting our project to get a review from a TSE
- Home page
  - When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
  - Along with the category name, you will display some numeric values per category that come from the API.
  - When the user clicks (or taps) on a category item, the application navigates to the details page.

- Details page
  - In the details page, the web app retrieves data from the API to show detailed data in the category.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.

**Testing requirements**
- Create unit tests for pure functions in your Redux code.
- Create integration tests for your application using the React Testing Library.
  - You may need to mock the access to the API so that your tests don't send actual requests.
  - You may need to mock the connection to the Redux Store.

**Technical requirements**

- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API should be stored in the Redux store.
- You should filter the data that you retrieve from the API using a Filter stateless component.
- Every page (the main page and the pages for each item) should have a unique route within the SPA.
- The project should be deployed and accessible online (add a link to the online version of your app to your README file).

### Project documentation

Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a **maximum length of 5 minutes**. The content of the video should include:

- a description of the project.
- a demo of the project features.
- you should also highlight some interesting piece of code or something you built that you are very proud of.

Happy review!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,84 +106,58 @@ Example:
...
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+- Netlifly for deploying
+🚩 Official Website [Netlifly](https://app.netlify.com/)
```
Comment (codepantha) [inline README.md]: - [ ] Please gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/components/Home/Home.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <ul className="categories">
+      {foods.length <= 1 ? 'Loading...'
+        : foods.map((food) => (
+          <li key={food.category_id} className="category">
+            <img src={food.category_image} alt={food.category_name} />
+            <h2>{food.category_name}</h2>
+            <NavLink to="seemore"><button id={food.category_id} type="button" onClick={handleJoin}>See More</button></NavLink>
+          </li>
+        )) }
+    </ul>
```
Comment (codepantha) [inline src/components/Home/Home.js]: - [ ] Please model the cards to look exactly like the given design guideline as shown below 👇 
- [ ] Make sure the card background colors are chequered to look like a chess board
- [ ] Kindly include the arrow icon at the top right of each card as shown below
## Design Guideline
image

 ## Your Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @Marcraphael12 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] At the moment, there is no way to filter through the data on the Home Screen. Please, kindly implement a Filter stateless component to filter data retrieved from the API as per requirements 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @Marcraphael12 , 

I am Favour 👩🏽 , your reviewer for this PR.

Great job implementing all the required changes suggested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Marcraphael12** [comment]
Thank you so much for approving 😁😁😁

**Pull request closed without merging**